modules/plugins/languages/r.nix

```
@@ -18,6 +18,37 @@
     packages = with pkgs.rPackages; [languageserver];
   };
 
+  defaultFormat = "format_r";
+  formats = {
+    styler = {
+      package = pkgs.rWrapper.override {
+        packages = with pkgs.rPackages; [styler];
```

Since we only have one package here, we can use omitting the scope.
```suggestion
        packages = [pkgs.rPackages.styler];
```